src/client/pages/OfferNew/Introduction/Sidebar/StartDate.tsx

```diff
@@ -296,7 +301,7 @@ const DateForm: React.FC<{
   const hasStartDate = Boolean(getDefaultDateValue(quote))
 
   return (
-    <RowButtonWrapper>
+    <RowButtonWrapper isSplit>
```

Comment:
Don't we need to pass down the prop here?

```suggestion
    <RowButtonWrapper isSplit={isSplit}>
```